File: raindrops/raindrops.go
package raindrops

import (
	"fmt"
)

func Convert(input int) string {
	result := ""
	if input%3 == 0 {
		result += "Pling"
	}
	if input%5 == 0 {
		result += "Plang"
	}
	if input%7 == 0 {
		result += "Plong"
	}
	if result == "" {
		result = fmt.Sprintf("%d", input)
	}
	return result
}

File: typescript/raindrops/raindrops.ts
export default class Raindrops {
  convert(n: number): string {
    let result = ""
    if (n % 3 === 0) {
      result += "Pling"
    }
    if (n % 5 === 0) {
      result += "Plang"
    }
    if (n % 7 === 0) {
      result += "Plong"
    }
    if (result.length === 0) {
      result = n.toString()
    }

    return result
  }
}